virt/support/virt-troubleshooting.adoc

```diff
@@ -37,42 +37,56 @@ You can view resource events by running the following command:
 $ oc describe <resource> <resource_name>
 ----
 
-[id="virt-logs_{context}"]
-== Logs
+[id="virt-pod-logs_{context}"]
+== Pod logs
 
-You can review the following logs for troubleshooting:
+You can view logs for {VirtProductName} pods by using the web console or the CLI. You can also view xref:../../virt/support/virt-troubleshooting.adoc#virt-viewing-logs-loki_virt-troubleshooting[aggregated logs] by using the LokiStack in the web console.
 
-* xref:../../virt/support/virt-troubleshooting.adoc#viewing-virt-component-pods_virt-troubleshooting[Virtual machine]
-* xref:../../virt/support/virt-troubleshooting.adoc#virt-viewing-logs-cli_virt-troubleshooting[{VirtProductName} pod]
-* xref:../../virt/support/virt-troubleshooting.adoc#virt-viewing-logs-loki_virt-troubleshooting[Aggregated {VirtProductName} logs]
+include::modules/virt-configuring-pod-log-verbosity.adoc[leveloffset=+2]
 
-include::modules/virt-viewing-virtual-machine-logs-web.adoc[leveloffset=+2]
+include::modules/virt-viewing-virt-launcher-pod-logs-web.adoc[leveloffset=+2]
 
-[id="viewing-virt-component-pods_{context}"]
-=== Viewing {VirtProductName} pod logs
+include::modules/virt-viewing-logs-cli.adoc[leveloffset=+2]
 
-You can view logs for {VirtProductName} pods by using the `oc` CLI tool.
+[id="guest-system-logs_{context}"]
+== Guest system logs
 
-You can configure the verbosity level of the logs by editing the `HyperConverged` custom resource (CR).
+Viewing the boot logs of VM guests can help diagnose issues. You can configure access to guests' logs and view them by using either the {product-title} web console or the `oc` CLI. 
 
-include::modules/virt-viewing-logs-cli.adoc[leveloffset=+3]
+This feature is disabled by default. If a VM does not explicitly have this setting enabled or disabled, it inherits the cluster-wide default setting.
 
-include::modules/virt-configuring-pod-log-verbosity.adoc[leveloffset=+3]
+[IMPORTANT]
+====
+If sensitive information such as credentials or other personally identifiable information (PII) is written to the serial console, it is logged with all other visible text. It is highly recommended to use SSH to transmit sensitive data instead of using the serial console for this purpose.
```

Comment:
- **issue:** Insted of `It is highly recommended to use SSH ...` please use `Red Hat recommends using SSH ...`.
- **nitpick:** We can also remove `for this purpose` without changing the meaning.